src/js/base/Context.js

```diff
@@ -54,6 +54,11 @@ export default class Context {
   }
 
   _initialize() {
+    // apply original placeholder
+    if (this.options.inheritPlaceholder && this.$note.attr('placeholder')) {
```

Comment:
if (this.options.placeholder === true) {
    this.options.placeholder = this.$note.attr('placeholder') || null;
}

Reply:
after a second thought, this changes should be added in placeholder module but not in global zone (Context.js)

Reply:
Agree. I will find another way to do this.